### ISSUE TITLE
Summer Plan to consolidate results into journal form

### ISSUE THREAD

**Issue by jonc101** [HealthRex/CDSS#250]
6/7/2019
	• Assembling data into batches so network can review them. E.g., log transforms and unit normalization
	• Bracket out next 5+ week plan?
	• [ ] GPU Config
	• [ ] Feature Selection (remove by variance and correlation)
	• [ ] Loss function definition
	• [ ] Train + Tune (at least start to reveal holes)

6/14/2019
	• Jon Wang out
	• [ ] Train + Tune
6/21/2019
	• Jon Wang out
	• [ ] Continue training while still improving
	• [ ] Order set prediction subset initiation
		○ Multinomial prediction still. Which order sets used within next 24 hours. Skip patient cases if no order sets used within 24 hours.
6/28/2019
	• Jon Wang out
	• [ ] Catch up on all prior steps
7/5/2019
	• Jon Wang out
	• [ ] Baselines: Logistic regression 
	• [ ] Human authored baseline
	• [ ] Order set prediction baseline (just use most common given admission diagnosis)
	• [ ] Evaluation metrics
7/12/2019
	• Jon Wang back in town
	• [ ] Evaluation Metrics - AUROC, PR Curve, Precision and Recall at K (K set at number of correct possible options R-Precision or based on human-authored benchmark)
7/19/2019
	• ??Error Analysis (better wait until after submission)
7/26/2019
8/3/2019
	• Med school starts

To Dos
	• Order Set Prediction data subset
	• ML baselines
		○ Logistic regression
	• Human authored baseline
		○ Order set
	• Order set prediction baseline
		○ Diagnosis alone
	• Feature selection
	• Train and tune network
		○ Training could take a long time. Track training loss as add batches and see when plateau
	• Setup GPUs
	• Process / Convert Test Patient Data
	• Evaluation metrics
		○ AUROC, PR Curve, Precision + Recall + F1 at K
	• Figures
	• Error / qualitative analysis
	• Manuscript Drafting
		○ Methods
		○ Results
		○ Reference Management
		○ Intro - Motivation / Objective
		○ Discussion - Core findings, limitations, conclusions
		○ Internal Review Editing x2
		○ Abstract
		○ Cover Letter (suggested reviewers, tie in to journal)
		○ Data source / code upload link
	• Journal Submission cycles x2-3
	• Codebase clean-up
	• Review and respond to journal comments
		○ Redo experiments
		○ Update figures
		○ Revise manuscript
		○ Resubmit

… Delaney and Jon Wang tracking these through Slack Weekly Updates
